webapp/packages/core-ui/src/Form/FormPart.ts

```diff
@@ -89,7 +89,10 @@ export abstract class FormPart<TPartState, TFormState = any> implements IFormPar
         return;
       }
 
-      await this.saveChanges();
+      await this.saveChanges(data, contexts);
+      if (ExecutorInterrupter.isInterrupted(contexts)) {
+        return;
```

Comment:
`this.loading` can stuck in `true` state then.

`this.loading = false;` - can be added here

Reply:
No, it can't it will be set to false in finally